lib/reversetunnel/localsite.go

```diff
@@ -104,8 +102,11 @@ type localSite struct {
 	// certificateCache caches host certificates for the forwarding server.
 	certificateCache *certificateCache
 
-	// remoteConns maps UUID and connection type to an remote connection.
-	remoteConns map[connKey]*remoteConn
+	// remoteConns maps UUID and connection type to remote connections, oldest to newest.
+	remoteConns map[connKey][]*remoteConn
+
+	// remoteConnsMtx protects remoteConns.
+	remoteConnsMtx sync.Mutex
```

Comment:
Should this be a `sync.RWMutex`?

Reply:
According to @fspmarshall any serious contention would be write contention as nodes try to connect at the same time, so a `RWMutex` wouldn't help things.

Reply:
👍